src/benchmarklib/benchmark_runner.hpp

```diff
@@ -41,6 +41,9 @@ class BenchmarkRunner {
 
   static nlohmann::json create_context(const BenchmarkConfig& config);
 
+  // If the query execution should be validated, this stores a pointer to the used SQLite instance
+  std::unique_ptr<SQLiteWrapper> sqlite_wrapper;
```

Comment:
any reason this is public (and other members aren't)? I am all for public members, as long as there is consistency in a class.

Reply:
Because there is no reason to touch them. We should only make members public if they would otherwise have an accessor.

Reply:
I think I was referring to sqlite_wrapper seemingly not being accessed from anyone else but the BenchmarkRunner, but this is a minor point, feel free to resolve.

Reply:
I can't follow. Who can't access it?

Reply:
Nevermind, I just saw that the index creation happens outside of the BenchmarkRunner, in main(). (I - erroneously - suspected the sqlite_wrapper was made public without reason.)